### PR TITLE
updated flashMessage feature

### PR DESCRIPTION
**Pull request by cmokbel1**
removed the return redundancy by defining variables that are mutable by the message being passed in. removed the setTimeout across the applicaiton for the messageState and moved it into the flashMessage component.

### CODE REVIEW

File: frontend/src/components/FlashMessage.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-            </div>
-        )
+        classes = "container alert alert-danger flash";
+        text = returnError;
     }
-
-}
+    return (
+        <div className={classes}>
+            <div className="text-center">
```
Comment (1gm) [inline frontend/src/components/FlashMessage.js:17]: Does adding 'text-center' to classes and removing this div work?

File: frontend/public/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="A React driven to do application"
```
[the remark this answers is not in the document]
Comment (cmokbel1) [reply frontend/public/index.html:10]: changed

File: frontend/src/components/FlashMessage.js (hunk 1 above)
Comment (cmokbel1) [reply frontend/src/components/FlashMessage.js]: yes

File: frontend/src/components/FlashMessage.js [hunk 3]
```diff
@@ -1,20 +1,22 @@
-export const FlashMessage = ({ messageState, returnError }) => {
+export const FlashMessage = ({ messageState, setMessageState, returnError }) => {
+    let text;
```
[the remark this answers is not in the document]
Comment (cmokbel1) [reply frontend/src/components/FlashMessage.js:2]: addressed

**Review by 1gm — APPROVED** [review]

**Pull request merged**